### PR TITLE
Added consent form to the configuration page

### PR DESCRIPTION
**Pull request by almakantara**
Added consent form to the configuration page

### CODE REVIEW

File: views/configure.pug [hunk 1]
```diff
@@ -26,5 +26,46 @@ block content
...
+  div.consent-form
+    p
+      | You are invited to participate in a research study on virtual assistants. With your consent, the Genie virtual assistant will record the commands you give it.
+      | Recording the commands will allow Genie to improve its understanding of natural language. The collection is completely confidential, and personal data (such as emails and phone numbers) will be stripped automatically.
+      | You must be 18 or older to participate, and currently residing in the United States.
+
+    h3 Risks and Benefits
+    p
+     | We do not foresee any risk from this study. The benefits which may reasonably be expected to result from this study are that the open-source Genie virtual assistant improves over time. We cannot and do not guarantee or promise that you will receive any benefits from this study.
+     //-| Your decision whether or not to participate in this study will not affect the functionality of the Genie service, or your ability to use it.
```
Comment (gcampax) [inline views/configure.pug:53]: This line should be uncommented for almond-server

File: views/configure.pug [hunk 2]
```diff
@@ -26,5 +26,46 @@ block content
...
       button(type='submit').btn.btn-primary Configure
+    
+
+
+mixin consent_form
+  h2=_("Consent Form")
+
+  div.consent-form
+    p
+      | You are invited to participate in a research study on virtual assistants. With your consent, the Genie virtual assistant will record the commands you give it.
```
Comment (gcampax) [inline views/configure.pug:46]: For now, in almond-server it should talk about the Almond virtual assistant, not Genie

File: views/configure.pug (hunk 1 above)
Comment (almakantara) [reply views/configure.pug:53]: fixed

File: views/configure.pug (hunk 2 above)
Comment (almakantara) [reply views/configure.pug:46]: fixed

**Review by gcampax — APPROVED** [review]
Awesome! Let's merge this

**Pull request merged**